sdk/nodejs/runtime/main.go

```diff
@@ -120,7 +129,9 @@ func (t *TypescriptSdk) Base(version string) *Container {
 
 	return dag.Container().
 		From(fmt.Sprintf("node:%s", version)).
-		WithMountedCache("/usr/local/share/.cache/yarn", dag.CacheVolume("mod-yarn-cache-"+version))
+		WithMountedCache("/usr/local/share/.cache/yarn", dag.CacheVolume("mod-yarn-cache-"+version)).
```

Comment:
Why keep this if not using yarn?

Reply:
Fixed in https://github.com/dagger/dagger/pull/6270/commits/691d72fb580b718d93cc5f1dcc88ce46e7d1389f